source/main.cpp

```diff
@@ -199,6 +204,10 @@ int main(int argc, char *argv[])
 	Screen::SetRaw(GameWindow::Width(), GameWindow::Height());
 	Preferences::Save();
 
+	// Load global Conditions:
+	DataWriter globCons(Files::Config() + "globalConditions.txt");
+	GameData::GlobalConditions().Save(globCons);
```

Comment:
For consistency I would expect global conditions to be written every time when regular conditions are also written to savegames. (within playerInfo.cpp).